src/blocks/recommendations/components/parsely-recommendations-fetcher.tsx

```diff
@@ -58,7 +58,7 @@ const ParselyRecommendationsFetcher = ( { boost, limit, sort, isEditMode } : Par
 		}
 
 		if ( error ) {
-			dispatch( setError( { error } ) );
+			dispatch( setError( { error: error as string } ) );
```

Comment:
is it an Error or a string?

```suggestion
			dispatch( setError( error ) );
```

Reply:
In the code at one place we are assigning type `unknown` to it and at one place type `string` ..... so just to make minimal changes I forced the typing here else we could have make changes at other places which I think doesn't like in the scope of this PR 🤔 

Implementing the suggested change is giving following error so leaving it as it is

```
Argument of type '{}' is not assignable to parameter of type 'SetErrorPayload'.
  Property 'error' is missing in type '{}' but required in type 'SetErrorPayload'.
```

Reply:
Ok, makes sense. Maybe a future PR could clean up error typing everywhere. Thanks for looking!